packages/react-button/src/components/ToggleButton/stories/ToggleButtonWithLongText.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { ToggleButton } from '../../../ToggleButton'; // codesandbox-dependency: @fluentui/react-button ^9.0.0-beta
+
+export const WithLongText = () => (
+  <>
+    <ToggleButton>Text</ToggleButton>
+    <ToggleButton>Text truncates after it hits the max width token value</ToggleButton>
```

Comment:
This is not actually truncating the text, but will merge anyway and we can fix it as a new item.